sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/models/EventRoute.java

```diff
@@ -1,15 +1,20 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
-// Code generated by Microsoft (R) AutoRest Code Generator.
 
 package com.azure.digitaltwins.core.models;
 
 import com.azure.core.annotation.Fluent;
+import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonProperty;
 
-/** The EventRoute model. */
+/**
+ * The EventRoute model. Event routes are used for defining where published telemetry gets sent to. As an example, an
+ * event route can point towards an Azure EventHub as a consumer of published telemetry.
+ */
+// This class exists so that the public APIs don't directly consume a generated type and so that we can avoid exposing a validate() method
```

Comment:
Srikanta had mentioned during the API review that there was some kind of a flag that we could set at autorest generation to get rid of this validation, did that not work?

Reply:
Even with the required args being set at constructor time (see autorest.md for this change), we still have the client side validation flag set to true. Because of that, we still get these validate() functions. I didn't want to get rid of the client side validation flag, so the only option I had was to use wrapper classes to cover up these validate() functions